ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/BehaviorTreeFileLoader.java

```diff
@@ -79,15 +93,16 @@ private T loadFromFile(JsonNode jsonNode, T parentNode, BehaviorTreeTopologyOper
          JsonNode fileNode = childJsonNode.get("file");
          if (fileNode == null)
          {
-            loadFromFile(childJsonNode, node, topologyOperationQueue);
+            loadFromFile(filePath, childJsonNode, node, topologyOperationQueue);
          }
          else
          {
             WorkspaceResourceFile childFile = new WorkspaceResourceFile(behaviorTreeState.getSaveFileDirectory(), fileNode.asText());
             LogTools.info("Loading {}", childFile.getFilesystemFile());
+            String relativePath = treeFilesDirectory.getFilesystemDirectory().relativize(childFile.getFilesystemFile()).toString();
```

Comment:
Needs testing. Also this should similarly load from classpath if not available from file, which would need testing.